lib/tasks/statistics.rake

```diff
@@ -112,3 +118,13 @@ namespace :statistics do
     puts output.to_json
   end
 end
+
+def dedup_nearby(activities)
+  last_activity = nil
+  activities.map do |activity|
+    if last_activity.nil? || !activity.very_similar_to(last_activity)
```

Comment:
I had a look around in the active record API, I couldn't see `very_similar_to`, where does it come from and what does it do?

Very useful if it's exactly what it says on the tin!

Reply:
It's defined in our models:

[`data_activity`](https://github.com/alphagov/govuk-account-manager-prototype/blob/e991a375fa38f77332f8f925a2c362b344069e17/app/models/data_activity.rb#L7)

[`security_activity`](https://github.com/alphagov/govuk-account-manager-prototype/blob/e991a375fa38f77332f8f925a2c362b344069e17/app/models/security_activity.rb#L60)